build-system/dep-check-config.js

```diff
@@ -169,6 +169,8 @@ exports.rules = [
         'src/service/video-manager-impl.js',
       'extensions/amp-fx-parallax/0.1/amp-fx-parallax.js->' +
           'src/service/parallax-impl.js',
+      'extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js->' +
```

Comment:
This is ok for now, but VERY VERY dangerous and should be refactored very soon. Please add `TODO()` with bug number and high priority.